module/proto/peggy/v1/genesis.proto

```diff
@@ -100,7 +100,7 @@ message Params {
   ];
   uint64  unbond_slashing_valsets_window           = 17;
   uint64  unbond_slashing_batch_window           = 18;
-
+  string inj_contract_address     = 19;
```

Comment:
Remove `inj_` prefix and change it to something generic. I suggest `cosmos_coin_erc20_contract`, basically denom of the main coin is a variable and can change a lot, so we could just reference to it as cosmos coin. Which in our case will have denom `inj` and specific erc20 address.